src/ui/pages/SwapForm/Quotes/useQuotes.ts

```diff
@@ -165,6 +166,25 @@ export function useQuotes({
     }
   }, [quote?.transaction]);
 
+  const swapViewRef = useRef(swapView);
+  swapViewRef.current = swapView;
+  useEffect(() => {
+    if (quote && done && swapViewRef.current) {
+      const { spendAsset, receiveAsset, spendPosition, store } =
+        swapViewRef.current;
+      walletPort.request('formQuoteReceived', {
```

Comment:
As we discussed, I would avoid any mentions of "form" in this hook. I updated it in my [bridge PR](https://github.com/zeriontech/zerion-wallet-extension/pull/724/files#diff-d745b187df88d997da5b687888cdc165098650c8455097b3eed1b113b649928e), so it no longer uses `SwapFormView`